lib/port.c

```diff
@@ -168,7 +168,7 @@ static struct port *getportent (void)
 	}
 	*cp = '\0';
 	cp++;
-	port.pt_names[j + 1] = NULL;
+	port.pt_names[j] = NULL;
```

Comment:
```
Reviewed-by: Alejandro Colomar <alx@kernel.org>
```

BTW, this reminds me of something I had pending.  I'll save that refactor for after the current one is merged.  :-)